javascript/javascript3/week3/homework-js3-week3/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main class="main-container">
+        <div class="screenshot-section">
```

Comment:
Good job while using clear naming to differentiate different sections. If you think it could improve readability, you could use the section HTML tag instead of div both in lines 14 and 28.